app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt

```diff
@@ -58,16 +59,17 @@ fun bindImage(imgView: ImageView, imgUrl: String?) {
 @BindingAdapter("marsApiStatus")
 fun bindStatus(statusImageView: ImageView, status: MarsApiStatus?) {
```

Comment:
Make status non nullable
`fun bindStatus(statusImageView: ImageView, status: MarsApiStatus)`

Reply:
this

Reply:
Done!